packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.types.ts

```diff
@@ -115,6 +115,12 @@ export interface IDatePickerProps extends IBaseProps<IDatePicker>, React.HTMLAtt
    */
   disableAutoFocus?: boolean;
```

Comment:
does the old prop need to be deprecated?

Reply:
No, it now does what it says it's going to do. It needs to be kept around!